gdbplus/gdb-12.1/gdb/gdb_dep.c

```diff
@@ -1623,6 +1623,121 @@ get_vtable_from_exp(const char*expression,
 	return rc;
 }
 
+/* Search and print types that matches the size range */
+void
+search_types_by_size(size_t min_sz, size_t max_sz)
```

Comment:
is this `search_types_by_size` one similar to the one in `gdbplus/gdb-9.2/gdb/gdb_dep.c`?
Having duplicate codes would mean we have to change more than two places if modifying this function.
I'm thinking if we can put all the common functions for ptmalloc into `src/heap_pt_malloc.cpp` which will not compiled as we are compiling src/heap_pt_malloc_2_27.cpp`.